packages/wrangler/package.json

```diff
@@ -101,6 +101,7 @@
 		"nanoid": "^3.3.3",
 		"path-to-regexp": "^6.2.0",
 		"selfsigned": "^2.0.1",
+		"source-map": "^0.7.4",
```

Comment:
we can move this to devDependencies because it'll get compiled into the final bundle

Reply:
We don't bundle it (included in `EXTERNAL_DEPNDENCIES`) due to the .wasm file not bundling correctly inside it

Reply:
oh that's right!